### PR TITLE
Artists

### PR DESCRIPTION
**Pull request by Sarahshr**
added artist carousel

### CODE REVIEW

**Comment by LoicBrassart** [comment]
Please assign me when the SM did her/his review

**Review by ThomasMlh — DISMISSED** [review]
It's ok ! 👍

**Review by Fx-Colliot — DISMISSED** [review]
yes

File: src/components/homepage/Artists.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  { width: 768, itemsToShow: 3 },
+  { width: 1200, itemsToShow: 4 },
+];
+
+const TrendArtists = () => {
+  const [data, setData] = useState([]);
+
+  const getMock = () => {
+    setData(mock.items);
+    return mock.items;
```
Comment (LoicBrassart) [inline src/components/homepage/Artists.jsx]: This return does nothing (you don't use the return value in your useEffect code)

File: src/components/homepage/Home.jsx [hunk 2]
```diff
@@ -1,16 +1,21 @@
 import React from "react";
 import TrendMusics from "./TrendMusics";
+<<<<<<< HEAD
```
Comment (LoicBrassart) [inline src/components/homepage/Home.jsx]: ACHTUNG !
There are conflict artefacts in this file !
Nobody read this code before me -_-

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Saw no Evil
@Fx-Colliot and @ThomasMlh , you didn't read this code: there are red blocks indicating bad code, it's utterly unmissable if you scroll this page. :anger: 
Next time I see something like this, I'll let the PR pass

**Review by Fx-Colliot — DISMISSED** [review]
ok je n'ai pas vu d'erreur

**Review by ThomasMlh — APPROVED** [review]
SnE !

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**